src/bun.js/bindings/bindings.zig

```diff
@@ -2414,6 +2414,19 @@ pub const JSGlobalObject = extern struct {
         return err;
     }
 
+    pub fn createError(
+        this: *JSGlobalObject,
+        code: JSC.Node.ErrorCode,
+        comptime error_name: string,
```

Comment:
error_name doesnt need to be comptime either

Reply:
oops